crates/sol-macro-gen/src/sol_macro_gen.rs

```diff
@@ -315,9 +322,13 @@ edition = "2021"
 
         let name_check = format!("name = \"{name}\"");
         let version_check = format!("version = \"{version}\"");
-        let alloy_dep_check = if let Some(version) = alloy_version {
+        let alloy_dep_check = if let Some(alloy_version) = alloy_version {
```

Comment:
this is the same logic as  the one below, should we externalize in a helper fn?

https://github.com/foundry-rs/foundry/blob/3f930e592e4421c096111d38b30ff5fa9fd0ff17/crates/sol-macro-gen/src/sol_macro_gen.rs#L131-L141

Reply:
Good catch, added in https://github.com/foundry-rs/foundry/pull/9861/commits/e8a921d3a5ec44a75eeecb7cddf2ef6b9931ecc7